radis/lbl/base.py

```diff
@@ -660,17 +660,29 @@ def get_Evib_CDSD_pcN_1iso(df, iso):
             # the map below is crazy fast compared to above loop
             df["Evibu"] = df_pcN.index.map(Evib_dict.get).values
 
-            return df.loc[idx, ["Evibl", "Evibu"]]
+            try:
+                return df.loc[idx, ["Evibl", "Evibu"]]
```

Comment:
Please have a look and carefully confirm, but I think idx is not defined in this function. It should be ":"  always, no need for try/except . 

Warning with try/except in general. It's OK at the user interface level but it's dangerous when changing  the internal physics part of the code like here

Reply:
Yes, you are right. I remember adding this because some tests fail; maybe I added it hereafter, adding it at some other place. Not sure. Will be careful with try/except while handling internal physics code henceforth.